client/src/components/_common/Sidebar/Sidebar.jsx

```diff
@@ -49,9 +49,20 @@ SidebarItem.defaultProps = {
   hidden: false,
 };
 
-const Sidebar = ({ sidebarItems, addItems, loading }) => {
+const Sidebar = ({ sidebarItems, addItemsBefore, addItemsAfter, loading, backgroundColor }) => {
   return (
-    <Nav className={styles['root']} vertical>
+    <Nav className={`${styles['root']}`} style={{background: backgroundColor}} vertical>
```

Comment:
**Nope 🙂.**

Please do not set CSS via `style` attribute. CSS is easier to find and more consistent to manage when it is in stylesheets.

Reply:
Noted!